internal/observability/event/sink_config.go

```diff
@@ -62,11 +63,22 @@ func (sc *SinkConfig) Validate() error {
 	if len(sc.EventTypes) == 0 {
 		return fmt.Errorf("%s: missing event types: %w", op, ErrInvalidParameter)
 	}
+
 	for _, et := range sc.EventTypes {
 		if err := et.Validate(); err != nil {
 			return fmt.Errorf("%s: %w", op, err)
 		}
+		// well, if there's an event type of audit, we need to check the audit
+		// config, if it's optionally provided.  We are intentionally only
+		// checking the FilterOverrides, because there's no way to specify the
+		// wrapper in a config.
```

Comment:
This brings up a good point. Would we want an embedded `kms` block? Or a single top-level block for type `audit`?

Doing it in an embedded way potentially means different sinks could use different KMSes. OTOH if we allow API-level configuration later then this just becomes the default.

Reply:
I'm leaning towards a single top-level audit kms block just for simplicity.  I sort of get the allure of the flexibilty that providing it the embedded way provides... I'm just not sure the added complexity is worth it... unless customers start asking for it in the future.

Reply:
OK -- in that case we can just add per-sink-level overrides later if we want.